test/javascripts/acceptance/docs-user-status-test.js

```diff
@@ -49,15 +50,15 @@ acceptance("Docs - user status", function (needs) {
 
   test("user status on mentions is live", async function (assert) {
     await visit("/docs?topic=1");
-    assert.dom(".mention .user-status").doesNotExist();
+    assert.notOk(exists(".mention .user-status-message"));
```

Comment:
Actually we are gradually moving from our helpers (like `exists()` here) to qunit-dom (the docs on testing need to be updated yet 😅)